go/service/account.go

```diff
@@ -260,3 +260,50 @@ func (h *AccountHandler) TimeTravelReset(ctx context.Context, arg keybase1.TimeT
 
 	return err
 }
+
+/////////////////////////
+type GetContactSettingsResponse struct {
+	libkb.AppStatusEmbed                                /// ?
+	Version              int                            `json:"version"`
+	AllowFolloweeDegrees bool                           `json:"allow_followee_degrees"`
+	Enabled              bool                           `json:"enabled"`
+	Teams                []keybase1.TeamContactSettings `json:"teams"`
+}
+
+func (h *AccountHandler) GetContactSettings(ctx context.Context, sessionID int) (ret keybase1.ContactSettings, err error) {
```

Comment:
don't know yet if i need sessionID

Reply:
you don't need a sessionID here since you don't need to "band" this RPC together with others.